cassandra-bridge/build.gradle

```diff
@@ -40,6 +40,7 @@ publishing {
 
 dependencies {
     api(project(':cassandra-analytics-common'))
+    api(project(':cassandra-analytics-spark-converter'))
```

Comment:
does this belong here? it feels the bridge should be spark agnostic. We should be able to take the bridge and use it elsewhere without having to bring the spark bits

Reply:
I think bridge is the connecting point between Cassandra and Spark. It already depends on the `spark-core`, `spark_sql`  jars.